src/components/SearchBar/__snapshots__/SearchBar.stories.storyshot

```diff
@@ -79,7 +79,7 @@ exports[`storyshots Storyshots Components/SearchBar Default 1`] = `
           placeholder="optional, regexp to search"
           required={true}
           spellCheck={false}
-          type="search"
+          type="text"
```

Comment:
It's annoying that we have to make this change 🙄 but I guess the warning is more annoying 😆

Reply:
yeah hmmm the placement of the search icon on the `SearchInput` makes it not so great for a one-to-one replacement 😔